version/version.go

```diff
@@ -0,0 +1,16 @@
+package version
```

Comment:
Go doesn't allow circular package dependencies. The logic was previously in `command` and needs to be reused in both `command` and `github`. So it has to be moved to one of the rest of the packages or a new one. I was thinking of a new `hub` package which stores hub specific stuff like main or version. But it requires bigger changes so I made the `version` package for now.